CMakeLists.txt

```diff
@@ -22,7 +22,10 @@ COMPONENTS
   cmake_modules
 )
 find_package(console_bridge REQUIRED)
-find_package(Eigen REQUIRED)
+# Eigen3 hack for Jade http://wiki.ros.org/jade/Migration#Eigen_CMake_Module_in_cmake_modules
+find_package(Eigen3)
+find_package(PkgConfig)
+pkg_search_module(Eigen3 REQUIRED eigen3)
```

Comment:
I don't understand how this hack is the same as the one in the link you [reference](http://wiki.ros.org/jade/Migration#Eigen_CMake_Module_in_cmake_modules), they look pretty different to me.